man/sigrok_rdm_sniffer.1

```diff
@@ -0,0 +1,48 @@
+.TH sigrok_rdm_sniffer 1 "March 2021"
+.SH NAME
+sigrok_rdm_sniffer \- Decode DMX/RDM data from devices supported by sigrok
```

Comment:
I'd replace `devices` with `logic sniffers` here

Reply:
I went for logic analyzer (with the US spelling) to match what sigrok call them in their docs.